.github/workflows/test-client.yml

```diff
@@ -69,6 +69,24 @@ jobs:
           echo "NETWORK=${{ inputs.network }}" >> $GITHUB_ENV
           echo "AGGREGATOR_ENDPOINT=${{ inputs.aggregator_endpoint }}" >> $GITHUB_ENV
           echo "GENESIS_VERIFICATION_KEY=$(curl -s ${{ inputs.genesis_verification_key }})" >> $GITHUB_ENV
+          echo "TRANSACTIONS_HASHES_TO_CERTIFY=${{ inputs.transactions_hashes_to_certify }}" >> $GITHUB_ENV
+
+      - name: Prepare environment variable for Cardano transactions aggregator capability (ubuntu-22.04, macos-12)
```

Comment:
`Assessing aggregator capabilities (Unix)` (btw you are not setting an environment variable here). Same kind of comment for the similar steps.